spotty/spotty/Data/Network/SpotifyAPI/SpotifyEndpoint.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  SpotifyEndpoint.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 10.06.22.
+//
+
+import Foundation
+
+enum SpotifyEndpoint {
+    
+    // MARK: - Cases
+    
+    case getCurrentUserProfile
+    case getCurrentUserTopTracks(limit: Int, offset: Int = 0)
```

Comment:
Maybe we can refactor here adding a parameter Enum ContentType with string cases tracks and artists.  Then here we will have only one case _getCurrentUserTop_ instead of 2 cases.